playlist/playlist.go

```diff
@@ -122,6 +140,21 @@ func (s *Store) Read(relPath string) (*Playlist, error) {
 			playlist.Comment = value
 		case attrIsPublic:
 			playlist.IsPublic, _ = strconv.ParseBool(value)
+		case attrSharedWith:
+			sharedWith := strings.Split(value, ",")
```

Comment:
Currently thinking whether it would be better for this to be a comma-separated list (as it is now), or if we should use the comment multiple times 🤔
```
#GONIC-NAME:"Roobre"
#GONIC-COMMENT:""
#GONIC-IS-PUBLIC:"false"
#GONIC-SHARED-WITH:"2"
#GONIC-SHARED-WITH:"3"
```
The latter sounds easier to parse.